.github/workflows/frontend-ci.yml

```diff
@@ -70,3 +70,24 @@ jobs:
             echo "Generated types are up-to-date."
             exit 0
           fi
+      - uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: migrations-changes
+        with:
+          filters: |
+            has-changes:
+              - '.github/workflows/frontend-ci.yml'
+              - 'frontend/packages/db/schema'
+              - 'frontend/packages/db/supabase/migrations'
+      - name: Check schema.sql
+        if: ${{ steps.migrations-changes.outputs.has-changes == 'true' }}
+        run: |
+          pnpm --filter @liam-hq/db schema:gen
```

Comment:
I'll integrate this command into pnpm --filter @liam-hq/db supabase:gen in a separate PR. 🙏

Reply:
#1025